packages/db/src/loaders/resources/networkGenealogies/index.ts

```diff
@@ -0,0 +1,2 @@
+export { AddNetworkGenealogies } from "./add.graphql";
+export { FindAncestors, FindDescendants } from "./search.graphql";
```

Comment:
I propose we just remove all the loaders changes, since you're not using them and I'm deleting them elsewhere :)

Reply:
You got it!